### PR TITLE
[DinoMod] Adding nothronychus to Dinoclysm

### PR DESCRIPTION
**Pull request by Drafting-Goblin**
#### Summary

SUMMARY: Content "This adds nothronychus to Dinoclysm"

#### Purpose of change

Nothronychus was one of the suggestions for issue #45162 

#### Describe the solution

This adds the hatchling, juvenile and adult forms of the nothronychus.
It also adds zombie forms for the juvenile and adult forms of the nothronychus, as well as a brute, shady and skeletal evolutions.

#### Describe alternatives you've considered

The biggest alternatives would have been not adding this dinosaur.
Smaller alternatives would be changing a few things in the PR. I'm not a paleontologist and this is the first creature that I've made for CDDA. I'm willing to explain my reasoning for anything, and change things about this if need be. 

#### Testing

I spawned each creature that this PR adds, fought and butchered them. Their eggs hatch into hatchlings. I've found a nothronychus in the wild.

#### Additional context

As stated, this is the first creature that I've added and currently the most complex PR I've done. I've taken extra care to cover all bases, but please let me know if I've missed something or if there is an area that could be improved. Thank you.

### CODE REVIEW

**Review by LyleSY — CHANGES_REQUESTED** [review]
This is ridiculously good for a first try.  Really good choices, you've clearly thought about it. I'd suggest bumping down the size and HP for the juveniles to match the other juvenile theropods and get rid of the zombie juvenile version (it will be too small to zombify if it copies the others). This is much bigger than a Deinonychus but it runs three times as fast. I can't quickly confirm that, can you point me in the right direction or slow it down? Last, can you add [DinoMod] to the start of your title? It helps out the change logger.

**Comment by Drafting-Goblin** [comment]
Thanks! That really means a lot. I've made the requested changes. Let me know if there's anything else that needs to be changed.

File: data/mods/DinoMod/monsters/juvenile.json [hunk 1]
```diff
@@ -223,6 +223,31 @@
...
   },
+  {
+    "id": "mon_nothronychus_juvenile",
+    "type": "MONSTER",
+    "name": "green and magenta two-legged juvenile",
+    "description": "A small bipedal plant-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
+    "default_faction": "herbivore_dino",
+    "copy-from": "mon_gallimimus_juvenile",
+    "color": "green_magenta",
+    "volume": "800 L",
```
Comment (LyleSY) [inline data/mods/DinoMod/monsters/juvenile.json:234]: So close! The juvenile still has the volume and weight of a full grown adult. Just removing those two lines will fix

**Review by LyleSY — CHANGES_REQUESTED** [review]
The volume and weight lines on the juvenile still need to be removed so it isn't the size and weight of an adult, but otherwise this is ready

**Comment by Theawesomeboophis** [comment]
I lowered weight and volume to be more appropriate.

**Comment by Theawesomeboophis** [comment]
Working on adding orcs to magiclysm and fixing Qianzhousaurus.

**Comment by Theawesomeboophis** [comment]
I fixed Qianzhousaurus as well, now just have to make baby and juvenile as well as zombie versions. It already spawns in the Lab or at least I told it to.

**Comment by Theawesomeboophis** [comment]
Also, what is the verdict on the frog, should I add it or no?

**Comment by Theawesomeboophis** [comment]
Separator not allowed glitch, might be because Qianzhousaurus isn't finished.

**Comment by LyleSY** [comment]
> I lowered weight and volume to be more appropriate.

Can you check again? I don't see that

File: data/mods/DinoMod/monsters/juvenile.json (hunk 1 above)
Comment (Drafting-Goblin) [reply data/mods/DinoMod/monsters/juvenile.json]: Thanks! Removed them, sorry about missing that last night.

**Review by LyleSY — APPROVED** [review]
This dino is ready to run

**Comment by Theawesomeboophis** [comment]
I sent the weight to him in a pull request. It's not on my fork.

**Comment by Drafting-Goblin** [comment]
That's odd, I didn't get any sort of notification for that. Thank you for helping.

**Comment by Theawesomeboophis** [comment]
I did it wrong anyways, accidentally gave juvenile hatchling weight, actual
weight of juvenile dinosaur is around 1800 kg.

**Pull request merged**